content/en/apps/guides/messaging/gateways/rapidpro_cht_gateway.md

```diff
@@ -0,0 +1,65 @@
+#
```

Comment:
Add [front matter](https://gohugo.io/content-management/front-matter/) with at least; title, linkTitle, and description.